src/utils/user.ts

```diff
@@ -0,0 +1,34 @@
+import axios from "api/axios"
+import { AxiosResponse } from "axios"
+import { initUser } from "api/profile"
+
+export async function setUser(): Promise<AxiosResponse> {
+	let response: AxiosResponse
+	response = await initUser()
```

Comment:
Instead of using api call in a util function try creating an Action for it in respected service file and using this util function to store in local storage there.

Reply:
done